source/guides/exchanges/exchange-REST-troubleshooting.rst

```diff
@@ -0,0 +1,218 @@
+   :local:
+
+**********************
+Receiving transactions
+**********************
+
+Filtering (by block height and more)
+====================================
+
+Use the `/transactions/confirmed <https://docs.symbolplatform.com/symbol-openapi/v1.0.1/#operation/searchConfirmedTransactions>`__ endpoint to retrieve any group of confirmed transactions. Among other filtering options, the following parameters are typically useful for narrowing down searches:
+
+.. csv-table::
+   :header: "Parameter", "Type", "Effect"
+   :delim: ;
+   :widths: 25 15 60
+
+   ``type``; integer; Retrieve only transactions of this type. The code for :ref:`Transfer Transactions <transfer-transaction>` is 16724 (0x4154).
+   ``recipientAddress``; string; "The address **receiving** the transaction. Compare with ``address`` which returns all transactions involving the given address (as sender, recipient or cosigner).
+   
+   This might either be a Base32 :ref:`address` or a :doc:`../../concepts/namespace`. If the bit 0 of byte 0 is not set (e.g. 0x90) then it is an address, otherwise (e.g. 0x91) it represents a namespace id which starts at byte 1."
+   ``fromHeight``; string; The starting block height to search for.
+   ``toHeight``; string; The ending block height to search for.
+   ``embedded``; boolean; "Use ``true`` to automatically search inside aggregate transactions. The data returned by a standalone transaction and an embedded transaction (one inside an aggregate) is slightly different (Check the next section).
+
+   Use ``false`` to manually search for aggregate transactions (e.g. types 0x4141 and 0x4241) and parse them.
+   
+   **Caution**: Does not work when combined with ``address``, only with ``recipientAddress``."
+
+`Here is an example query <http://ngl-dual-101.testnet.symboldev.network:3000/transactions/confirmed?recipientAddress=TAOXUJOTTW3W5XTBQMQEX3SQNA6MCUVGXLXR3TA&type=16724&fromHeight=70000&toHeight=80000&embedded=true>`__:
+
+.. code-block:: text
+
+   http://ngl-dual-101.testnet.symboldev.network:3000
+     /transactions/confirmed
+       ?recipientAddress=TAOXUJOTTW3W5XTBQMQEX3SQNA6MCUVGXLXR3TA
+       &type=16724
+       &fromHeight=70000
+       &toHeight=80000
+       &embedded=true
+
+This query returns the list of all **transfer** transactions sent to address ``TAOXU...``, that happened between block height **70000** and **80000**, including transactions **embedded** inside other transactions.
+
+The next section explains how to parse the resulting list.
+
+Parsing of embedded transactions
+================================
+
+As shown in the `/transactions/confirmed <https://docs.symbolplatform.com/symbol-openapi/v1.0.1/#operation/searchConfirmedTransactions>`__ endpoint documentation, a successfull query returns **a data array** including transactions and metadata. Each one of the returned transactions can match a different schema depending on the transaction's type, so the ``type`` field of each transaction must be checked.
```

Comment:
The transaction endpoint actually returns a set of paginated transactions which should be mentioned here I think. If the pagination parms are not provided, default values (page size / number) will be used instead.

Reply:
Added a note!